FreeRTOS/FreeRTOS-Plus-TCP
# Changes to incorporate UDP changes for IPv6

UDP protocol support for IPv6

Description
-----------
UDP protocol support for IPv6
The UDP functionality is split into 2 files FreeRTOS_IPV4.c and FreeRTOS_IPV6.c to support both IPv4 and IPv6.

Test Steps
-----------
As of now compilation test is being done.

cmake -S test/build-combination -B test/build-combination/build/ -DTEST_CONFIGURATION=ENABLE_ALL
make -C test/build-combination/build/


Related Issue
-----------
Unit tests will be validated once all the changes are merged. Hence, this might break some unit tests in the dev branch.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
